python_modules/dagit/dagit/webapp/src/Config.tsx

```diff
@@ -20,8 +20,13 @@ function renderTypeRecursive(
     return (
       <>
         {`{`}
-        {type.fields.map((fieldData: any) => (
+        {type.fields.map(fieldData => (
           <DictEntry key={fieldData.name}>
+            {fieldData.description && (
+              <DictComment>{`${innerIndent}/* ${
```

Comment:
I still think for short field-level comments we should figure out a way to display both the field declaration and the field on a single line. I also think we can probably get away with just changing the font and not having to wrap the comments in C-style comment blocks

Reply:
Oh, hmm - I suppose we can do them inline if they're less than a specific length. The sidebar is fairly narrow though and I expect anything more than 3-4 words will wrap.

I think we could also push users to stop adding descriptions that add so little value over the field name + type, though. We can say `database: <String> // Database name` in one line, but it'd really be better to just call the field database_name and remove the description entirely, saving descriptions for things that actually need an explanation.)